packages/services/server/src/index.ts

```diff
@@ -604,6 +605,15 @@ export async function main() {
       targetsByIdCache: registry.injector.get(TargetsByIdCache),
     });
 
+    server.post('/cache/organization-access-token-cache/delete/:id', async (req, res) => {
```

Comment:
very useful for integration tests to check if membership permission/resource updates propagate